<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="My test suite" duration-ms="3310" started-at="2018-08-03T07:17:55Z" finished-at="2018-08-03T07:17:59Z">
    <groups>
    </groups>
    <test name="单元测试" duration-ms="3310" started-at="2018-08-03T07:17:55Z" finished-at="2018-08-03T07:17:59Z">
      <class name="testng.ApiTestTestNg">
        <test-method status="PASS" signature="parameterIntTest(api.Coverage)[pri:0, instance:testng.ApiTestTestNg@dfd3711]" name="parameterIntTest" duration-ms="1959" started-at="2018-08-03T07:17:57Z" data-provider="parameter" finished-at="2018-08-03T07:17:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Coverage [loginname=abc, loginpass=abc]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parameterIntTest -->
        <test-method status="FAIL" signature="parameterIntTest(api.Coverage)[pri:0, instance:testng.ApiTestTestNg@dfd3711]" name="parameterIntTest" duration-ms="1964" started-at="2018-08-03T07:17:57Z" data-provider="parameter" finished-at="2018-08-03T07:17:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Coverage [loginname=1212, loginpass=1111]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Lists differ at element [0]: true != false expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Lists differ at element [0]: true != false expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:831)
at org.testng.Assert.assertEquals(Assert.java:796)
at testng.ApiTestTestNg.parameterIntTest(ApiTestTestNg.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parameterIntTest -->
      </class> <!-- testng.ApiTestTestNg -->
    </test> <!-- 单元测试 -->
  </suite> <!-- My test suite -->
</testng-results>
